CHANGELOG.md

```diff
@@ -212,6 +216,20 @@ Implemented a proper way of caching dependency repositories. The cache is stored
 
 * `viash test`: Fix an issue where the tests would not copy package config settings to determine the docker image name (PR #767).
 
+# Viash 0.8.7 (2025-04-01): Backport support upcoming version of Nextflow
```

Comment:
if Viash 0.8.8 is released first, then we can also add the 0.8.8 changelog to this PR.

Reply:
I added the 0.8.8 changelog entry

Reply:
Thanks!